### PR TITLE
Update your code

### PR DESCRIPTION
**Pull request by PrathmeshAgrawal**

### CODE REVIEW

File: README [hunk 1]
```diff
@@ -1,19 +1,19 @@
 The Steam protocol plugin for bitlbee. This plugin uses the Steam Mobile
 API allowing it to run alongside the main Steam client. It is worth
-noting that the Steam Mobile API is HTTP based, which does lead to mild
+noting that the Steam Mobile API is HTTP based plugin, which does lead to mild
```
Comment (dgw) [inline README:3]: Doesn't make sense.
```suggestion
noting that the Steam Mobile API is HTTP based, which does lead to mild
```

File: README [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 latency.
 
 ## Package Repositories
 
 There are package repositories available for various distributions.
 
-See: https://jgeboski.github.io
+See: https://jgeboski.github.io  For further details
```
Comment (dgw) [inline README:10]:
```suggestion
See https://jgeboski.github.io for further details
```

File: README [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
 
 ## Building and Installing
 
-Make sure bitlbee and its headers have been installed. If bitlbee came
+Make sure, bitlbee and its headers have been installed. If bitlbee came
```
Comment (dgw) [inline README:14]: The original syntax is correct.

```suggestion
Make sure bitlbee and its headers have been installed. If bitlbee came
```

File: README [hunk 4]
```diff
@@ -69,7 +69,7 @@ Output game play statues to the account channel(s):
 
 ## Common Issues
 
-Below is a list of common issues and their respective fixes.
+Below is a list of common issues and their respective fixes. Follow these steps to fix these issues
```
Comment (dgw) [inline README:72]: Kinda redundant, and missing a period anyway.

```suggestion
Below is a list of common issues and their respective fixes.
```

**Comment by chauffer** [comment]
@jgeboski this seems like [hacktoberfest](https://hacktoberfest.digitalocean.com/faq/) spam.

>We dislike seeing spam pull requests just as much as you, so please give them an invalid or spam label and close them. Pull requests that contain a label with the word invalid or spam won’t be counted toward Hacktoberfest.

File: README [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
-See: https://jgeboski.github.io
+See: https://jgeboski.github.io  For further details
 
 ## Building and Installing
 
-Make sure bitlbee and its headers have been installed. If bitlbee came
+Make sure, bitlbee and its headers have been installed. If bitlbee came
 from the distribution's repository, it will most likely need the
-development package, usually bitlbee-dev.
+development package which you must install, usually bitlbee-dev.
```
Comment (dgw) [inline README:16]: This is redundant with the previous part of that clause ("will most likely need" ≈ "you must install").

```suggestion
development package, usually bitlbee-dev.
```

**Review by dgw — COMMENTED** [review]
Account created October 1, 2020… I'm guessing this user joined GitHub for Hacktoberfest. Agree with @chauffer that it would be best to tag this as invalid and close it. Most of the changes (annotated below) are regressions to the wording, not improvements.

**Comment by dequis** [comment]
lmao

**Pull request closed without merging**